pdns/recursordist/docs/settings.rst

```diff
@@ -1066,6 +1066,31 @@ Path to a lua file to manipulate the Recursor's answers. See :doc:`lua-scripting
 The interval between calls to the Lua user defined `maintenance()` function in seconds.
 See :ref:`hooks-maintenance-callback`
 
+.. _setting-max-busy-dot-probes:
+
+``max-busy-dot-probes``
+-----------------------
+.. versionadded:: 4.7.0
+
+- Integer
+- Default: 0
+
+Limit the maximum number of simultaneous DoT probes the Recursor will schedule.
+The default value 0 means no DoT probes are scheduled.
+
+DoT probes are used to check if an authoritative server's IP address supports DoT.
+If the probe determines an IP address supports DoT, the Recursor will use DoT to contact it for subsequent queries until a failure occurs.
+After a failure, the Recursor will stop using DoT for that specific IP address for a while.
```

Comment:
As mentioned by someone somewhere, _a while_ isn't very helpful.

Reply:
This is under-documented on purpose, as these things will change once the code becomes less experimental.